collect_app/src/main/res/values/strings.xml

```diff
@@ -654,4 +654,6 @@
     <string name="image_not_saved">The image can not be saved because of a problem with the corresponding instance file.</string>
     <string name="loading_form_failed">An error occurred while loading the form. Please try again.</string>
     <string name="sms_invalid_phone_number_description">Set a destination phone number for SMS submissions or change your transport type.</string>
+    <string name="google_invalid_account_title">Google account missing!</string>
```

Comment:
No exclamation mark. I know we have some in other text but should probably reduce them because it's not something many apps tend to do.